pom.xml

```diff
@@ -25,6 +26,9 @@
 
   <properties>
     <java.version>1.8</java.version>
+    <kafka.version>2.4.1</kafka.version>
+    <slf4j.version>1.7.7</slf4j.version>
+    <log4j.version>1.2.17</log4j.version>
```

Comment:
If this module is going to be opinionated about logging, I'd strongly suggest not using the legacy log4j framework.
Drop in the latest log4j 2.x and disruptor, and configure asynchronous loggers (as opposed to asynchronous appenders).
https://logging.apache.org/log4j/2.x/manual/async.html

Reply:
Created separate JIRA for this.

Reply:
*Issue 😅

Reply:
this is in the root pom, not the kafka-streams module